src/Features/Core/Portable/FindUsages/SourceReferenceItem.cs

```diff
@@ -22,7 +22,7 @@ internal sealed class SourceReferenceItem
         /// <summary>
         /// The location of the source item.
         /// </summary>
-        public DocumentSpan SourceSpan { get; }
+        public SerializableDocumentSpan SourceSpan { get; }
```

Comment:
this is the otehr part that ensures we're not rooting the solution if someone holds onto a RefItem.